charts/kubewarden-controller/templates/opentelemetry-collector.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       batch:
     exporters:
       {{- if .Values.telemetry.tracing.jaeger.endpoint }}
-      jaeger:
+      otlp/jaeger:
         endpoint: {{ .Values.telemetry.tracing.jaeger.endpoint }}
```

Comment:
There is no `jaeger` field any more in the values. It's changed to `otlp/jaeger`. Thus, we need to change these references to it as well.

Reply:
updated values.yaml with `make generate-values` (argh).
I'm not sure I understand this comment,  do you mean changing `.Values.telemetry.tracing.jaeger.` to `.Values.telemetry.tracing.otlp_jaeger.` or similar?

Reply:
Yes. Otherwise, if the user change the endpoint value it will never be used in the collector configuration. Because there is no `jaeger` field anymore.

Reply:
Another option, is just use the whole field content in the collector config. Therefore, we do not need to map all the configuration available to the exporter

Reply:
> Another option, is just use the whole field content in the collector config. Therefore, we do not need to map all the configuration available to the exporter

still, we have the exporter that we need to change. 


After talking,  I force-pushed a commit that only changes the template, but not the values. Given that whatever we do, if we provide the OpentelemetryCollector resource, we get burned either way.